packages/expo-modules-core/scripts/generateKSPLookup.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
+
+const minKotlinVersion = '1.6.10';
+const maxKotlinVersion = '2.0.21';
+
+const groupId = 'com.google.devtools.ksp';
+const artifactId = 'symbol-processing-gradle-plugin';
+const path = 'expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/KSPLookup.kt';
```

Comment:
```suggestion
const path = require('path').resolve(__dirname, '../expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/KSPLookup.kt');
```

in case people run the script in the scripts directory or other places